### PR TITLE
Bump version to 6.1.2

### PR DESCRIPTION
**Pull request by ajainarayanan**
```[INFO] CDAP CLI ........................................... SUCCESS [  5.620 s]
[INFO] CDAP Gateway ....................................... SUCCESS [  2.446 s]
[INFO] CDAP UI ............................................ SUCCESS [04:31 min]
[INFO] CDAP Operational Stats Core ........................ SUCCESS [  2.105 s]
[INFO] CDAP Google Cloud Dataproc Runtime Extension ....... SUCCESS [  3.339 s]
[INFO] CDAP Amazon EMR Runtime Extension .................. SUCCESS [  0.617 s]
[INFO] CDAP Remote Hadoop Runtime Extension ............... SUCCESS [  0.338 s]
[INFO] CDAP Google Cloud KMS Secure Store Extension ....... SUCCESS [  0.363 s]
[INFO] CDAP Standalone .................................... SUCCESS [ 16.808 s]
[INFO] CDAP Java Client Tests ............................. SUCCESS [  2.942 s]
[INFO] CDAP CLI Tests ..................................... SUCCESS [  0.593 s]
[INFO] CDAP Integration Test Framework .................... SUCCESS [  0.754 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  08:01 min
[INFO] Finished at: 2020-03-24T20:19:46-07:00
[INFO] ------------------------------------------------------------------------```

### CODE REVIEW

**Comment by yaojiefeng** [comment]
Is this just pom change? I think there are other places. Take a look at https://github.com/cdapio/cdap/pull/11963

File: cdap-tms/pom.xml [hunk 1]
```diff
@@ -161,7 +161,7 @@
     <dependency>
       <groupId>io.cdap.cdap</groupId>
       <artifactId>cdap-storage-spi</artifactId>
-      <version>6.1.2-SNAPSHOT</version>
```
Comment (CuriousVini) [inline cdap-tms/pom.xml:164]: not from this PR, but this should use `${project.version}`

**Review by CuriousVini — APPROVED** [review]
one comment. rest lgtm

File: cdap-tms/pom.xml (hunk 1 above)
Comment (ajainarayanan) [reply cdap-tms/pom.xml:164]: Done.

**Review by CuriousVini — APPROVED** [review]
lgtm

**Review by yaojiefeng — APPROVED** [review]

**Comment by ajainarayanan** [comment]
Green build: https://builds.cask.co/browse/CDAP-DRC9916/latest. Merging.

**Pull request merged**